api/src/Page/Proposal.php

```diff
@@ -393,7 +393,8 @@ function _get_visits($visit = null, $output = true)
         }
 
         if ($this->has_arg('s')) {
-            $where .= " AND s.visit_number LIKE :" . (sizeof($args) + 1);
+            $where .= " AND (s.visit_number LIKE :" . (sizeof($args) + 1) . " OR s.beamlinename LIKE :" . (sizeof($args) + 2) . ")";
+            array_push($args, $this->arg('s'));
             array_push($args, $this->arg('s'));
```

Comment:
Is this meant to be duplicated?

Reply:
Yeah, line 396 now has 2 references to $args so you have to add the search term to $args twice. For some reason you can't refer to the same position in $args twice, so you end up repeating lines like this, or looping like on lines 184-5 of the same file.